### ISSUE TITLE
object-literal-key-quotes changes broke object shorthand

### ISSUE THREAD

**Issue by corbinu** [palantir/tslint#1938]
### Bug Report

- __TSLint version__: `4.2.0`
- __TypeScript version__: `2.1.4`
- __Running TSLint via__: npm script/CLI

#### TypeScript code being linted

```ts
let obj = {
     event,
     tags
};
```

#### Actual behavior

PR #1874 causes this to be treated as a unquoted object literal

#### Expected behavior

no error

**Comment by adidahiya** [comment]
Fixed by #1945

**Comment by corbinu** [comment]
Thanks!
